sdk/spring/azure-spring-cloud-core/src/main/java/com/azure/spring/core/properties/retry/BackoffProperties.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.core.properties.retry;
+
+/**
+ * Backoff properties when a retry fails.
+ */
+public class BackoffProperties {
+
+    private long delay;
```

Comment:
we should use wrapper class instead of primitive type, as a best practice, to avoid unexpected default value

Reply:
Got it, will modify in following PR.